context.go

```diff
@@ -3071,6 +3071,22 @@ func (c *Context) BlueprintFile(logicModule Module) string {
 	return module.relBlueprintsFile
 }
 
+func (c *Context) LogicModules() []Module {
```

Comment:
This is already available through ctx.VisitAllModules.

Reply:
Ahh, of course! Thank you, this PR is not needed then :)